dotcom-rendering/docs/patterns/enhance-capi.md

```diff
@@ -2,23 +2,23 @@
 The enhance CAPI pattern is one where the CAPI object is transformed into a different, more useable, format. In particular, it relates to the `blocks` array. Each enhancement function, takes the `CAPIType` and returns the `CAPIType`.
 
 ## Lexicon
-An 'enhancement' could also be termed a 'transformation' and in other places similar functions are refrered to as 'cleaners'.
+An 'enhancement' could also be termed a 'transformation' and in other places similar functions are refereed to as 'cleaners'.
 
 ## Why?
 Ideally, these transformations would not be required. A better solution would be to construct the array at source, in Composer or CAPI, with the ideal structure. But for - probably very good - reasons the decision was taken to use [cleaners in frontend](https://github.com/guardian/frontend/blob/aa0013a6f9c247be36d29b9716e0ccc80cc8b218/common/app/views/support/HtmlCleaner.scala) to solve design problems. So now, in order to support the same designs, we need to replicate these cleaners in DCR. The enhancements are applied in the [server/index.ts](/dotcom-rendering/src/web/server/index.ts).
 
 ## Examples
 
 ### DropCaps
-Certain article types, such as Features, have the first letter of the first paragraph marked as a DropCap. However, there was a requirement to sometimes also have other paragraphs be given a DropCap. But composer does not offer a way to mark a paragraph as, say, dropcap: true, so instead a convention was invented where if the preceeding element was a h2 tag containing '* * *' then that was the trigger to give the following paragraph drop cap styling.
+Certain article types, such as Features, have the first letter of the first paragraph marked as a DropCap. However, there was a requirement to sometimes also have other paragraphs be given a DropCap. But composer does not offer a way to mark a paragraph as, say, `dropcap: true`, so instead a convention was invented where if the preceding element was a h2 tag containing '* * *' then that was the trigger to give the following paragraph drop cap styling.
```

Comment:
```suggestion
Certain article types, such as Features, have the first letter of the first paragraph marked as a DropCap. However, there was a requirement to sometimes also have other paragraphs be given a DropCap. But composer does not offer a way to mark a paragraph as, say, `dropcap: true`, so instead a convention was invented where, if the preceding element was an `h2` tag containing '* * *', then that was the trigger to give the following paragraph drop cap styling.
```

Reply:
I did wonder about this one. It's _a_ heading, but I would deffo read it as H-2.